Description: disable version check which referred to pgplot5, and we're using giza
Origin: vendor
Forwarded: not-needed
Author: Ole Streicher <olebole@debian.org>
Reviewed-by: gregor herrmann <gregoa@debian.org>
Last-Update: 2021-11-21

--- a/t/t12.t
+++ b/t/t12.t
@@ -13,7 +13,6 @@
 
 PGPLOT::pgqinf("VERSION", my $val, my $len);
 $val =~ s/\.//g; $val =~ s/v//;
-plan skip_all => "PGPLOT version must be > 5.2.0 for this test $val\n" if $val<520;
 
 # Read in image (int*2)
 
